ouroboros/__builtins__.py

```diff
@@ -135,6 +135,30 @@ def __exit__(self, type, value, tb):
 
 sentinel = object()
 
+#min
```

Comment:
There doesn't appear to be any purpose for this comment.

Reply:
Removed